kernel/aster-nix/src/process/process/timer_manager.rs

```diff
@@ -76,7 +74,7 @@ fn update_cpu_time() {
 
 /// Registers a function to update the CPU clock in processes and
 /// threads during the system timer interrupt.
-pub(super) fn init() {
+pub fn timer_manager_init() {
```

Comment:
The recommended convention is to keep exactly one initialization method named `init` to do any kind of initialization work for a module. From the parent module's perspective, it should not be concerned as to why a sub-module needs to do initialization. So calling it `init` is enough. Furthermore, a module may need to do more initialization as the module grows more complex. Thus, giving its initialization function the most general name `init` is extensible for future needs.

So please revert the changes to `process::init` and `timer_manager::init`.

Reply:
`timer_manager::init` need to be called in the thread context while `process::posix_thread::futex::init` needs to be called way earlier. I explicitly separated them and removed `process::init`.